.editorconfig

```diff
@@ -175,7 +175,7 @@ csharp_style_throw_expression = true:error
 csharp_style_var_elsewhere = true:suggestion
 
 # http://kent-boogaart.com/blog/editorconfig-reference-for-c-developers#csharp_style_var_for_built_in_types
-csharp_style_var_for_built_in_types = false:error
+csharp_style_var_for_built_in_types = true
```

Comment:
Can you explain please?

Reply:
this is a linter rule that doesn't conform with our coding style, and already a lot of the project has lint errors because of it.
I disabled it for now at least.